packages/core/src/dispatcher.ts

```diff
@@ -163,7 +170,8 @@ export class Dispatcher {
 
   async getIpfsBlock(cid: CID): Promise<Uint8Array> {
     return await this._shutdownSignal.abortable((signal) => {
-      return this._ipfs.block.get(cid, { signal })
+      // @ts-ignore
+      return this._ipfs.block.get(cid, { signal, offline: !this._enableSync })
```

Comment:
Where did you find info about this `offline` option? Maybe we should include a reference in a comment for context?

Reply:
From a message from Adin in the filecoin slack.  Also realized we've used it before: https://github.com/ceramicnetwork/js-ceramic/blob/81afa77edeedb8e69f5e74c44dfccd85e8052f35/packages/core/src/dispatcher.ts#L274

Reply:
ok!